.github/workflows/dev-cluster-check.yml

```diff
@@ -1,6 +1,19 @@
 name: Check development cluster
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
I think this may be useful also for other cluster checks like stable etc. Would it be possible to add these inputs also to more workflows?

Reply:
sure

Reply:
@jiripetrlik done